### PR TITLE
feat: add vscode-html-languageservice parser converter to data dictionary

### PR DESCRIPTION
**Pull request by janechu**
# Description

This adds a utility to convert the data coming from the `vscode-html-languageservice` parser into a `DataDictionary` format used by the message system.

## Motivation & context

This will be used in the Monaco Adaptor as a means to convert changes from the Monaco editor to the Message System data format.

Begins work on #3562

## Issue type checklist

- [ ] **Chore**: A change that does not impact distributed packages.
- [ ] **Bug fix**: A change that fixes an issue, link to the issue above.
- [x] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

**Adding or modifying component(s) in `@microsoft/fast-components` checklist**

- [ ] I have added a new component
- [ ] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

## Process & policy checklist

- [x] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### CODE REVIEW

**Comment by janechu** [comment]
Putting a hold on this as unexpectedly there may be an issue with using the `vscode-html-languageservice` in browser.

File: packages/tooling/fast-tooling/package.json [hunk 1]
```diff
@@ -98,5 +98,9 @@
     },
     "peerDependencies": {
         "lodash-es": "^4.0.0"
+    },
+    "dependencies": {
+        "@microsoft/fast-web-utilities": "^4.6.0",
+        "vscode-html-languageservice": "^3.1.2"
```
Comment (janechu) [inline packages/tooling/fast-tooling/package.json:104]:
```suggestion
        "vscode-html-languageservice": "^3.1.3"
```

**Comment by janechu** [comment]
Ready for review, this was solved in the `vscode-html-languageservice` v3.1.3 so I've fixed it to that version.

**Review by EisenbergEffect — APPROVED** [review]
Woohoo! 🥳

**Review by awentzel — APPROVED** [review]

**Pull request merged**